### PR TITLE
Correct price in auction details

### PR DESCRIPTION
**Pull request by josojo**
Price should be quoteToken per baseToken, i.e. 2000 DAI per WETH

### CODE REVIEW

File: src/components/auction/AuctionDetails/index.tsx [hunk 1]
```diff
@@ -303,7 +303,7 @@ const AuctionDetails = (props: AuctionDetailsProps) => {
                 }}
               />
               <TokenValue>
-                {abbreviation(derivedAuctionInfo?.initialAuctionOrder?.sellAmount.toSignificant(2))}
+                {abbreviation(derivedAuctionInfo?.initialAuctionOrder?.sellAmount.toSignificant(4))}
```
Comment (josojo) [inline src/components/auction/AuctionDetails/index.tsx:306]: closes #442

I am pretty sure that this is the fix

Comment (josojo) [reply src/components/auction/AuctionDetails/index.tsx:306]: @jarcodallo

**Review by gabitoesmiapodo — APPROVED** [review]
Looks good.

We should change this on the featured auctions cards too, but that can be fixed later.

**Review by mariano-aguero — APPROVED** [review]
LGTM

**Review by jarcodallo — APPROVED** [review]

**Pull request merged**